packages/plugin-ethereum/src/child/EthereumChildPlugin.ts

```diff
@@ -142,6 +129,7 @@ export class EthereumChildPlugin extends ChildPlugin<EthereumChildPluginConfig>
         throw new Error("no injected web3 provided");
       }
     } else if (providerType === ProviderTypes.PORTIS) {
+      const Portis = await import("@portis/web3");
```

Comment:
this will get rid of all those Portis error logs you see even if not using Portis, right?

Reply:
correct